src/frame.rs

```diff
@@ -147,21 +151,21 @@ impl FrameAllocator {
 /// assert_eq!(num, Some(0));
 /// ```
 #[cfg(feature = "use_spin")]
-pub struct LockedFrameAllocator(Mutex<FrameAllocator>);
+pub struct LockedFrameAllocator<const ORDER: usize = 32>(Mutex<FrameAllocator<ORDER>>);
 
 #[cfg(feature = "use_spin")]
-impl LockedFrameAllocator {
+impl<const ORDER: usize> LockedFrameAllocator<ORDER> {
     /// Creates an empty heap
-    pub fn new() -> LockedFrameAllocator {
+    pub fn new() -> Self {
         LockedFrameAllocator(Mutex::new(FrameAllocator::new()))
```

Comment:
use `Self` in place of `LockedFrameAllocator`

Reply:
Done.